packages/dd-trace/src/proxy.js

```diff
@@ -111,23 +111,36 @@ class Tracer extends NoopProxy {
     return this
   }
 
+  _enable (id, ...args) {
+    const module = require(id)
+    module.enable(...args)
+    this[id] = true
+  }
+
+  _disable (id) {
+    if (this[id]) {
+      const module = require(id)
```

Comment:
it could avoid requiring the module when calling _disable if we use a Map to store required modules (that was my first approach)